### DISCUSSION TITLE
can I use "langchain" in pyscript?? if not, I request the python package "langchain"

### DISCUSSION THREAD

**Discussion by cosmosanalytics** [pyscript/pyscript#1329] [Proposals]
Package Name and Version : langchain 0.0.125
Package URL : https://pypi.org/project/langchain/

**Comment by JeffersGlass** [comment]
It doesn't look like there are major impediments to this working in Pyodide; you'll probably want to [open a new package request in the Pyodide repo](https://github.com/pyodide/pyodide/issues).
